### PR TITLE
send event after tx broadcast

### PR DESCRIPTION
**Pull request by girazoki**

### CODE REVIEW

File: wallet/src/types.rs [hunk 1]
```diff
@@ -238,6 +238,8 @@ pub enum Event {
     /// A list of hashes of blocks that are now considered orphaned.
     BlocksOrphan(Vec<String>),
     /// A new movement (transaction) affecting balance.
+    LocalTransactionSent(model::BalanceMovement),
```
Comment (tmpolaczyk) [inline wallet/src/types.rs:241]: Should we create a new event, or should we reuse the existing `Movement`?

Comment (girazoki) [reply wallet/src/types.rs:241]: Depends on whether it makes a difference for sheikah

File: wallet/src/actors/app/methods.rs [hunk 2]
```diff
@@ -631,25 +631,37 @@ impl App {
...
         let f = fut::result(
             self.state
                 .get_wallet_by_session_and_id(&session_id, &wallet_id),
         )
-        .and_then(move |wallet, act: &mut Self, _| {
-            act.send_inventory_transaction(transaction.clone())
-                .and_then(move |jsonrpc_result, _slf, _ctx| {
+        .and_then(move |wallet, slf: &mut Self, _| {
+            slf.send_inventory_transaction(transaction.clone())
+                .and_then(move |jsonrpc_result, slf: &mut Self, _ctx| {
```
Comment (tmpolaczyk) [inline wallet/src/actors/app/methods.rs:644]: Let's try to follow the same convention as in the node

```suggestion
                .and_then(move |jsonrpc_result, act, _ctx| {
```

File: wallet/src/actors/app/methods.rs [hunk 3]
```diff
@@ -631,25 +631,37 @@ impl App {
...
                     match wallet.add_local_movement(&model::ExtendedTransaction {
-                        transaction,
+                        transaction: transaction.clone(),
```
Comment (tmpolaczyk) [inline wallet/src/actors/app/methods.rs:646]: The clone shouldn't be needed?

```suggestion
                        transaction,
```

**Review by tmpolaczyk — APPROVED** [review]

**Review by mariocao — APPROVED** [review]

**Pull request merged**